README.md

```diff
@@ -64,6 +64,38 @@ operators][operators].
 
 [Giant Swarm operators][giantswarm-operators] using operatorkit.
 
+
+
+## Integration Tests
+
+You can simply create a [`kind`](https://github.com/kubernetes-sigs/kind/)
+cluster to run the integration tests.
+
+```
+kind create cluster
+```
+
+The tests need to figure out how to connect to the Kubernetes cluster. Therefore
+we need to set an environment variable pointing to your local kube config.
+
+```
+export E2E_KUBECONFIG=<your-path>/.kube/config
```

Comment:
```suggestion
export E2E_KUBECONFIG=~/.kube/config
```

This should work for MacOS and Linux in most situations. If you're using Windows you've got bigger problems. :trollface: